### ISSUE TITLE
Browser support

### ISSUE THREAD

**Issue by chrahunt** [mafintosh/is-my-json-valid#70]
Are there any plans to include browser support in this project? IMJV performs far better than the current JSON Schema validators that also work in the browser (Z-Schema and tv4). Using browserify in combination with

```javascript
window.imjv = require('./index.js')
```

worked fine for me, and validating my objects takes 30ms compared to upwards of 3.5s using Z-Schema. I'd hate to think that others are skipping this library and its performance benefits because there isn't a pre-built version for web browsers.

**Comment by mafintosh** [comment]
Assuming its easy to maintain I'm :+1:

**Comment by chrahunt** [comment]
Any preference for global variable name? Maybe `IMJValidator`?

**Comment by mafintosh** [comment]
I trust your judgement :)

**Comment by sathishk** [comment]
+1 

I think it is not just speed. compared to other libraries (tv4, jsen), is-my-json-valid is good with complex validations. Ref: https://github.com/bugventure/jsen/issues/38

This schema works as expected in is-my-json-valid.

Browser support will be really helpful to get this to plain HTML applications also.

**Comment by mafintosh** [comment]
If anyone wants to do a PR for this I'd merge it

**Comment by Basdub** [comment]
Is it a complicated task to make it available to the browser? just asking, I have no idea ;)

Is it coming?

**Comment by mafintosh** [comment]
@Basdub you can use in the browser today with browserify. like i mentioned above its pretty easy to do static builds. we just need someone to add a script to package.json that will do that for us so we don't have to do it manually every time we publish a new version

**Comment by Basdub** [comment]
Yep get that. You'll laugh but I'm not Node.js ready/setup yet. @chrahunt , could you provide your script to add to package.json? 

Thanks

**Comment by Basdub** [comment]
Or, @chrahunt, could I get your browserified version of it? Thanks

**Comment by mafintosh** [comment]
@Basdub https://wzrd.in/standalone/is-my-json-valid@latest (window.isMyJsonValid(...) afterwards)

**Comment by igl** [comment]
Having it work in Browserify & Webpack is already fine for me, but you might want to checkout out UMD for this: https://github.com/umdjs/umd

**Comment by bugventure** [comment]
@sathishk The [JSEN issue](https://github.com/bugventure/jsen/issues/38) you've referred to actually represents a badly defined schema, not really a limitation of JSEN.
